### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by SamuelTakyiBoateng**
## BookStore (Styling)
- Added missing Jsx elements.
- Styled UI to conform to Zeplin design.

### CODE REVIEW

**Review by ridwanediallo — CHANGES_REQUESTED** [review]
# Changes Required ♻️
## Hi @SamuelTakyiBoateng , 
AlmostThereReachGIF

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- You used GitHub flow correctly. ✔
- All linters checks are passing as expected. ✔


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.css [hunk 1]
```diff
@@ -1,7 +1,415 @@
...
+
+nav ul li:first-child {
+  width: 40%;
+}
+
+.bookstore-cms {
+  width: 100%;
+  height: 2.313rem;
+  margin: 0.375rem 2.938rem 0.125rem 0;
+  font-family: sans-serif;
```
[the remark this answers is not in the document]
Comment (SamuelTakyiBoateng) [reply src/index.css]: Thank you so much.
Please, changes have been effected. Thanks again!

File: src/components/Form.jsx [hunk 2]
```diff
@@ -34,39 +34,48 @@ const Form = () => {
...
+            <option value="Comics">Comics</option>
+            <option value="Classic">Classic</option>
+            <option value="Thriller">Thriller</option>
+            <option value="Thriller">Romantic</option>
+          </select>
+        </label>
+        <button type="button" className="add-btn" onClick={handleAddClick}>
+          ADD BOOK
+        </button>
+      </div>
```
[the remark this answers is not in the document]
Comment (SamuelTakyiBoateng) [reply src/components/Form.jsx]: Thank you so much.
Please, changes have been effected. Thanks again!

**Review by Shedrack-Sunday — APPROVED** [review]
Hi @SamuelTakyiBoateng

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉

image


Highlights
- [x] Good project organisation ✔️
- [x] Nice use of React features ✔️  
- [x] Kudus on meeting the requirements  as suggested by the last reviwer ✔️

### Confirmed view:
image

Optional suggestions
Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**